blkn/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@kiwicom/balkan-ui",
-  "version": "0.0.0-rc0",
+  "version": "0.0.0-rc2",
   "scripts": {
-    "storybook": "start-storybook -p 6007 -c .storybook"
+    "storybook": "start-storybook -p 6007 -c .storybook",
+    "build": "babel --out-dir lib --ignore *.stories.js,*.test.js src",
```

Comment:
how about building also an `es` version? see the Kiwi part for an example

Reply:
Updated. Please review again.